### PR TITLE
Better message on OTP InternalError

### PR DESCRIPTION
**Pull request by alain-andre**

### CODE REVIEW

File: wrappers/otp.rb [hunk 1]
```diff
@@ -63,12 +63,17 @@ def route(locs, dimension, departure, arrival, language, with_geometry, options
...
+
+        begin
+          request = RestClient.get(@url + '/otp/routers/' + @router_id + '/plan', {
+            accept: :json,
+            params: params
+          })
+          data = JSON.parse(request)
+          @cache.write(key, data) if data && !data['error'] && data['plan'] && data['plan']['itineraries']
+        rescue RestClient::InternalServerError => ex
+          raise RouterWrapper::NoRouteFound.new(ex), "OTP, #{ex.message} on #{@router_id}"
```
Comment (fab-girard) [inline wrappers/otp.rb]: I suggest you: 
```
rescue Restclient::ExceptionWithResponse => e
  e.message += " - OTP #{@router_id}"
  raise e
```
not to overload original exception and message

Comment (alain-andre) [reply wrappers/otp.rb]: OK. done.

File: wrappers/otp.rb [hunk 2]
```diff
@@ -63,12 +63,18 @@ def route(locs, dimension, departure, arrival, language, with_geometry, options
...
-        @cache.write(key, data) if data && !data['error'] && data['plan'] && data['plan']['itineraries']
+
+        begin
+          request = RestClient.get(@url + '/otp/routers/' + @router_id + '/plan', {
+            accept: :json,
+            params: params
+          })
+          data = JSON.parse(request)
+          @cache.write(key, data) if data && !data['error'] && data['plan'] && data['plan']['itineraries']
+        rescue RestClient::ExceptionWithResponse => ex
```
Comment (fab-girard) [inline wrappers/otp.rb]: rubocop should explain it's bad not to name exception `e`:
https://www.rubydoc.info/gems/rubocop/0.75.0/RuboCop/Cop/Naming/RescuedExceptionsVariableName

Comment (alain-andre) [reply wrappers/otp.rb]: Yes, I changed it and I will push a PR to add rubocop.

**Review by fab-girard — APPROVED** [review]

**Pull request merged**